### ISSUE TITLE
[Breadth Coverage] Onboarding BlockChain Commands

### ISSUE THREAD

**Issue by MyronFanQiu** [Azure/azure-cli#11828]
Is your feature request related to a problem? Please describe.
no.

Describe the solution you'd like
Support Azure Blockchain service
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/blockchain/resource-manager

Describe alternatives you've considered

Additional context

**Comment by MyronFanQiu** [comment]
planned in S164/S165

**Comment by MyronFanQiu** [comment]
@yonzhan Hi. Should we close this issue first? It might violate the OKR 3.2 later. we can open a new issue once the swagger is ready later.

**Comment by yonzhan** [comment]
Add feature request and service attention.
